src/components/search/site-search-form.tsx

```diff
@@ -10,15 +10,15 @@ const SiteSearchForm = ({...props}: ComponentProps<any>) => {
           Search this site
         </label>
         <input
-          className="rounded-full h-20 w-full lg:w-100 text-20"
+          className="rounded-full h-15 w-full lg:w-100 text-15 lg:border-black-20"
           type="text"
           placeholder="Search this site"
           id={inputId}
           name="q"
           required
         />
-        <button type="submit" className="absolute top-5 right-5">
-          <MagnifyingGlassIcon width={30}/>
+        <button type="submit" className="absolute top-2 right-5">
+          <MagnifyingGlassIcon width={30} className="text-digital-red" />
```

Comment:
I think the size of the icon might need to shrink. It doesn't look proportionate and when you focus, the bounding box overlays the input area.
![image](https://github.com/SU-SWS/cardinalsites-nextjs/assets/7185045/847ad9bf-54fe-40b2-a3e1-09f3b6d0f6d1)

Reply:
I noticed that now when I fixed my zoom in from my no glasses day.